### ISSUE TITLE
vmalert has two default port

### ISSUE THREAD

**Issue by yimenghua** [VictoriaMetrics/operator#658]
internal]/config/config.go
VMAlertDefault.Port : 8080

api/v1beta1/vmalert_types.go
```go
func (cr *VMAlert) AsURL() string {
	port := cr.Spec.Port
	if port == "" {
		port = "8880"
	}
	return fmt.Sprintf("http://%s.%s.svc:%s", cr.PrefixedName(), cr.Namespace, port)
}
```

8080 or 8880 ?

**Comment by f41gh7** [comment]
Hello, it's a bug. Thx for pointing. It affects only service discovery for vmauth.

**Comment by f41gh7** [comment]
Fixed at [v0.35.0](https://github.com/VictoriaMetrics/operator/releases/tag/v0.35.0) release
